atomsci/ddm/utils/struct_utils.py

```diff
@@ -455,7 +453,7 @@ def fix_moe_smiles(smiles):
     """
     protn_pat = re.compile(r'\[([cCBnNPS])([-\+])(@*)(H[1234]*)*\]')
     scalar = False
-    if type(smiles) == str:
+    if isinstance(smiles, str):
```

Comment:
test?

Reply:
Do you have any MOE smiles to test this with?

Reply:
@mcloughlin2 might. I don't exactly know where MOE smiles would be created in the pipeline. During featurization?

Reply:
It's not called anywhere inside AMPL. This might be vestigial?